flink-cdc-runtime/src/main/java/org/apache/flink/cdc/runtime/operators/schema/coordinator/SchemaRegistryRequestHandler.java

```diff
@@ -437,6 +444,25 @@ private List<SchemaChangeEvent> lenientizeSchemaChangeEvent(SchemaChangeEvent ev
         }
     }
 
+    private boolean shouldIgnoreException(Exception exception) {
+
+        // only UnsupportedSchemaChangeEventException maybe ignore(depends on SchemaChangeBehavior)
```

Comment:
Code flow and comments are confusing here. Something like

```java
// Only `UnsupportedSchemaChangeEventException` could be ignored in TRY_EVOLVE mode.
// In IGNORE mode, will never try to apply schema change events
// In EVOLVE mode, such failure will not be tolerated
// In EXCEPTION mode, an exception will be thrown once captured
return (exception instanceof UnsupportedSchemaChangeEventException) && 
    (schemaChangeBehavior == TRY_EVOLVE)
```

might be enough.

Reply:
done it